README.md

```diff
@@ -209,6 +209,41 @@ static class WebConfig extends WebSecurityConfigurerAdapter {
 }
 ```
 
```

Comment:
@bdemers Should we reference https://developer.okta.com/docs/guides/customize-tokens-returned-from-okta/add-custom-claim/ here?

Reply:
I think I'm missing something here.  You are creating an Authorization server in the example?

Reply:
Nope, I am not creating an AS in the example.